package.json

```diff
@@ -8,8 +8,8 @@
     "test": "test"
   },
   "scripts": {
-    "test": "gulp test",
-    "start": "gulp watch"
+    "test": "./node_modules/.bin/mocha test/*",
```

Comment:
Well, it's not a big deal but regarding the fact that running `npm test` in developement is a common scenario - it makes sense to keep it clean and simple as it was earlier 👍